libr/anal/class.c

```diff
@@ -1238,12 +1238,13 @@ R_API RGraph *r_anal_class_get_inheritance_graph(RAnal *anal) {
 		r_graph_free (class_graph);
 		return NULL;
 	}
-	// O(1) lookup cache for processed nodes
-	// to lookup parents to create edges
+	/* O(1) lookup cache for processed nodes
```

Comment:
i dont get why this comment is split in 2 lines. apart that the comment can be removed because its quite obvious by the code to know what it is for. can you make it one line with // or remove it?

thanks!

Reply:
Yeah, removed the useless comments, lot of them are leftovers, I've kept some comments so someone can better orient between parts of the function if necessary